src/components/Button/button.module.scss

```diff
@@ -27,7 +27,7 @@
     --button-hover-variant-background-color: var(--primary-color-20);
     --text-color: var(--text-inverse-color);
     background-color: inherit;
-    border: 2px solid rgba(var(--button-primary-default-border-color), 0.01);
+    border: 8px solid rgba(var(--button-primary-default-border-color), 0.01);
```

Comment:
An 8px border is too thick.

Reply:
This will cause a noticeable flicker on hover and will be jarring.

Reply:
This PR is just to test chromatic 😅

Reply:
Ah, oops.